iree_tests/configs/config_sdxl_cpu_llvm_task.json

```diff
@@ -0,0 +1,17 @@
+{
+    "config_name": "sdxl_cpu_llvm_task",
+    "iree_compile_flags" : [
+      "--iree-hal-target-backends=llvm-cpu",
+      "--iree-llvmcpu-target-triple=x86_64-linux-gnu",
+      "--iree-llvmcpu-target-cpu-features=host",
+      "--iree-llvmcpu-fail-on-out-of-bounds-stack-allocation=false",
+      "--iree-llvmcpu-distribution-size=32"
```

Comment:
Are these flags _needed_ for these models? Prefer to use default flags for tests.

Reply:
Looks like the second one is, but removed the first one